src/Simplex/Chat/Library/Subscriber.hs

```diff
@@ -762,54 +762,68 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
           XOk -> pure ()
           _ -> messageError "INFO from member must have x.grp.mem.info, x.info or x.ok"
         pure ()
-      CON _pqEnc -> unless (memberStatus m == GSMemRejected) $ do
-        status' <- case memberStatus m of
-          GSMemPendingApproval -> pure GSMemPendingApproval
-          _ -> do
-            withStore' $ \db -> do
-              updateGroupMemberStatus db userId m GSMemConnected
-              unless (memberActive membership) $
-                updateGroupMemberStatus db userId membership GSMemConnected
-            -- possible improvement: check for each pending message, requires keeping track of connection state
-            unless (connDisabled conn) $ sendPendingGroupMessages user m conn
-            pure GSMemConnected
+      CON _pqEnc -> unless (memberStatus m == GSMemRejected || memberStatus membership == GSMemRejected) $ do
+        -- TODO [knocking] send pending messages after accepting?
```

Comment:
why removed?

Reply:
it's reverted to how it was before - I think it was a bug from when we added acceptance that this also got inside the condition, because it's then never sent anywhere.